src/Main.java

```diff
@@ -1,5 +1,62 @@
+import org.w3c.dom.ls.LSOutput;
+
+import java.time.LocalDate;
+
 public class Main {
+    public static boolean determineYear(int year) {
+        boolean yearLeap;
+        yearLeap = (((year % 4 == 0) && (year % 100 != 0)) || (year % 400 == 0));
+        return yearLeap;
+    }
+
+    public static void determineMessage(int yearDevice, String modelDevice) {
+
+        if (yearDevice < 2015) {
```

Comment:
```suggestion
        if (yearDevice < LocalDate.now().getYear()) {
```
по заданию надо сравнивать с текущим годом, а не 2015